src/utils/string/jiraLinkify.tsx

```diff
@@ -5,11 +5,11 @@ import { getJiraTicketUrl } from "constants/externalResources";
 export const jiraLinkify = (
   unlinkified: string | React.ReactNodeArray,
   jiraHost: string,
-  onClick?: (match: string) => () => void
+  onClick: () => void = () => {}
```

Comment:
if the prop is required it does not need a default value. we should make this optional again.